src/components/AreaView/AreaView.jsx

```diff
@@ -1,10 +1,16 @@
 import { React, useState, useEffect } from 'react';
 import { Link, Navigate, useParams } from 'react-router-dom';
 import { Button, Card } from 'react-bootstrap';
-import { TLPBackend } from '../../common/utils';
+import { BsPeople } from 'react-icons/bs';
+import { TLPBackend, calculateScores } from '../../common/utils';
 import DropdownMenu from '../../common/DropdownMenu/DropdownMenu';
 import SitesTable from './sitesTable';
 import styles from './AreaView.module.css';
+import NavigationBar from '../NavigationBar/NavigationBar';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'NavigationBar' is defined but never used.